project/Dependencies.scala

```diff
@@ -6,39 +6,268 @@ object Dependencies {
+      "javax.validation" % "validation-api" % "1.1.0.Final",
+      "joda-time" % "joda-time" % "2.9.6",
+      "junit" % "junit" % "4.11",
+      "net.java.dev.jna" % "jna" % "4.0.0",
+      "net.jodah" % "typetools" % "0.4.4",
+      "net.jpountz.lz4" % "lz4" % "1.3.0",
+      "oauth.signpost" % "signpost-commonshttp4" % "1.2.1.2",
+      "oauth.signpost" % "signpost-core" % "1.2.1.2",
+      "org.agrona" % "agrona" % "0.9.2",
+      "org.antlr" % "ST4" % "4.0.8",
+      "org.antlr" % "antlr" % "3.5.2",
+      "org.antlr" % "antlr-runtime" % "3.5.2",
+      "org.apache.cassandra" % "cassandra-all" % CassandraAllVersion,
+      "org.apache.cassandra" % "cassandra-thrift" % CassandraAllVersion,
+      "org.apache.commons" % "commons-lang3" % "3.4",
+      "org.apache.commons" % "commons-math3" % "3.2",
+      "org.apache.httpcomponents" % "httpclient" % "4.2.5",
+      "org.apache.httpcomponents" % "httpcore" % "4.2.4",
+      "org.apache.kafka" % "kafka-clients" % "0.10.0.1",
+      "org.apache.thrift" % "libthrift" % "0.9.2",
+      "org.apache.tomcat" % "tomcat-servlet-api" % "8.0.33",
+      "org.caffinitas.ohc" % "ohc-core" % "0.4.3",
+      "org.codehaus.jackson" % "jackson-core-asl" % "1.9.2",
+      "org.codehaus.jackson" % "jackson-mapper-asl" % "1.9.2",
+      "org.eclipse.jdt.core.compiler" % "ecj" % "4.4.2",
+      "org.fusesource" % "sigar" % "1.6.4",
+      "org.hibernate" % "hibernate-validator" % "5.2.4.Final",
+      "org.hibernate.javax.persistence" % "hibernate-jpa-2.1-api" % "1.0.0.Final",
+      "org.immutables" % "value" % "2.3.2",
+      "org.javassist" % "javassist" % "3.20.0-GA",
+      jbossLogging,
+      "org.joda" % "joda-convert" % "1.8.1",
+      "org.hamcrest" % "hamcrest-core" % "1.3",
+      "org.mindrot" % "jbcrypt" % "0.3m",
+      "org.pcollections" % "pcollections" % "2.1.2",
+      "org.reactivestreams" % "reactive-streams" % "1.0.0",
+      "org.reflections" % "reflections" % "0.9.10",
+      "org.scalactic" %% "scalactic" % ScalaTestVersion,
+      "org.scalatest" %% "scalatest" % ScalaTestVersion,
+      "org.scala-lang.modules" %% "scala-java8-compat" % ScalaJava8CompatVersion,
+      "org.scala-lang.modules" %% "scala-parser-combinators" % "1.0.4",
+      scalaXml,
+      "org.scala-sbt" % "test-interface" % "1.0",
+      "org.scala-stm" %% "scala-stm" % "0.7",
+      "org.springframework" % "spring-beans" % "4.2.7.RELEASE",
+      "org.springframework" % "spring-context" % "4.2.7.RELEASE",
+      "org.springframework" % "spring-core" % "4.2.7.RELEASE",
+      "org.uncommons.maths" % "uncommons-maths" % "1.2.2a",
+      "org.xerial.snappy" % "snappy-java" % "1.1.2.6",
+      "org.yaml" % "snakeyaml" % "1.16",
+      "tyrex" % "tyrex" % "1.0.1",
+      "xerces" % "xercesImpl" % "2.11.0",
+      "xml-apis" % "xml-apis" % "1.4.01"
+
+    ) ++ libraryFamily("com.fasterxml.jackson.core", JacksonVersion)(
+      "jackson-annotations", "jackson-core", "jackson-databind"
```

Comment:
Yeah, it might be easier if we want to be able to copy and paste output from a failed build, but then again it would also make it easier to mistakenly mix versions within a family.

Your call then.